src/main/java/org/jenkinsci/plugins/workflow/job/views/GraphVizAction.java

```diff
@@ -74,7 +74,7 @@ private GraphVizAction(WorkflowRun run) {
     public HttpResponse doDot() throws IOException {
         StringWriter sw = new StringWriter();
         writeDot(new PrintWriter(sw));
-        return HttpResponses.plainText(sw.toString());
+        return HttpResponses.text(sw.toString());
```

Comment:
So IIUC, the difference is that the new method flushes instead of appending a newline? Did you check via the UI to make sure the output still looks ok? It's probably fine, but good to be careful.

Reply:
> So IIUC, the difference is that the new method flushes instead of appending a newline?

Yeah, that is my understanding as well.

> Did you check via the UI to make sure the output still looks ok? It's probably fine, but good to be careful.

I added `r.interactiveBreak();` to the end of `WorkflowRunTest#culprits`, went to `http://${JENKINS_URL}/jenkins/job/p/2/graphViz/`, and got [this image](https://i.ibb.co/bBwkg6d/graphViz.png).

Reply:
Great, thanks for checking!